src/runtime_src/tools/xclbin/SectionPartitionMetadata.h

```diff
@@ -23,6 +23,7 @@
 #include "Section.h"
 #include "FDTProperty.h"
 #include <boost/functional/factory.hpp>
+#include <boost/property_tree/json_parser.hpp>
```

Comment:
Including headers from other headers should be kept to a minimum.   I don't believe that this jason_parser.hpp header is needed.

Reply:
this header is required for write_json(). Moved the include from .h to .cxx